core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -161,6 +161,31 @@ convert_errno(int error)
     return errors[error];
 }
 
+static bool
```

Comment:
Is it OK if assume `list` is an array of domain names and domain patterns?, like `[["*", "example", "com"], ["www", "google", "com"]]`. 

If so, which function should split the input string "*.example.com" -> ["", "" ,""]

Reply:
`list` is an array of all the values domain names or domain patterns (i.e. list of strings), so for example running `iwasm` with the following parameters:
```
iwasm --allow-resolve=*.example.com --allow-resolve=domain.com
```
will result with having two elements in the array: `["*.example.com", "domain.com"]`. For now we allow for very simplistic pattern matching where wildcard can be used as a prefix, so it can only appear as a first character of the string.